.github/workflows/aws-lambda-java-serialization.yml

```diff
@@ -28,9 +28,9 @@ jobs:
     # Install base module
     - name: Build events with Maven
       run: mvn -B install --file aws-lambda-java-events/pom.xml
-    # Install target module
+    # Package target module
     - name: Build serialization with Maven
```

Comment:
nit: Looking at this I think we should rename the workflow steps in a way that more accurately depicts what the workflow is doing with the module - eg. `s/Build/Package` and `s/Build/Install` when appropriate.